Ninano/Ninano/CalendarCollectionViewCell.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  CalendarCollectionViewCell.swift
+//  Ninano
+//
+//  Created by Yoonjae on 2022/07/17.
+//
+
+import UIKit
+
+class CalendarCollectionViewCell: UICollectionViewCell {
+
+    @IBOutlet weak var dateLabel: UILabel!
```

Comment:
```suggestion
    @IBOutlet private weak var dateLabel: UILabel!

    func update(date: String) {
        dateLabel.text = date
    }
```


https://github.com/DeveloperAcademy-POSTECH/MC3-Team17-Hou/pull/19/files#diff-c6d911237423366cb365baf297fd18ba0b3a4effe8454e04751ad59a68c691a0R104 의 코드를 줄이고, property도 private로 지킬 수 있다고 생각되어요 ~

Reply:
https://github.com/DeveloperAcademy-POSTECH/MC3-Team17-Hou/pull/19/files#diff-c6d911237423366cb365baf297fd18ba0b3a4effe8454e04751ad59a68c691a0R109-R115 Color 값도 함께 고민된다면 좋겠네용